internal/http/services/owncloud/ocs/data/capabilities.go

```diff
@@ -236,6 +236,7 @@ type CapabilitiesFilesSharingPublic struct {
 	ExpireDate         *CapabilitiesFilesSharingPublicExpireDate `json:"expire_date" xml:"expire_date" mapstructure:"expire_date"`
 	CanEdit            ocsBool                                   `json:"can_edit" xml:"can_edit" mapstructure:"can_edit"`
 	Alias              ocsBool                                   `json:"alias" xml:"alias"`
+	DefaultPermissions int                                       `json:"default_permissions" xml:"default_permissions" mapstructure:"default_permissions"`
```

Comment:
Did you check the Acceptance Criteria `Investigate: does a default link permission of 0 conflict with a default password policy? permission 0 links must always be created without password.` ?

Reply:
Yes, an internal link can be created without password, even when passwords are enforced 👍